app/src/main/java/org/p2p/wallet/auth/interactor/restore/UserRestoreInteractor.kt

```diff
@@ -119,7 +119,9 @@ class UserRestoreInteractor(
         if (web3AuthError.errorType == Web3AuthErrorResponse.ErrorType.CANNOT_RECONSTRUCT) {
             RestoreUserResult.UserNotFound
         } else {
-            RestoreUserResult.RestoreFailed(Throwable("Unknown error type"))
+            RestoreUserResult.SharesDoesNotMatch
```

Comment:
maybe we can specify which shares are no match? or it's for every restore case?

Reply:
I presume this is for every case when shares does not match and we can't do anything